src/comm/aca_comm_mgr.cpp

```diff
@@ -129,11 +139,16 @@ int Aca_Comm_Manager::update_goal_state(GoalState &goal_state_message,
   }
 
   auto end = chrono::steady_clock::now();
-
+  auto dhcp_operation_time =
+          cast_to_microseconds(end - neighbor_update_finished_time).count();
   auto message_total_operation_time = cast_to_microseconds(end - start).count();
 
-  ACA_LOG_INFO("[METRICS] Elapsed time for message total operation took: %ld microseconds or %ld milliseconds\n",
-               message_total_operation_time, us_to_ms(message_total_operation_time));
+  ACA_LOG_INFO("[METRICS] Elapsed time for message total operation took: %ld microseconds or %ld milliseconds\n[METRICS] Elapsed time for router operation took: %ld microseconds or %ld milliseconds\n[METRICS] Elapsed time for port operation took: %ld microseconds or %ld milliseconds\n[METRICS] Elapsed time for neighbor operation took: %ld microseconds or %ld milliseconds\n[METRICS] Elapsed time for dhcp operation took: %ld microseconds or %ld milliseconds\n",
```

Comment:
Please change this log to `ACA_LOG_DEBUG`

Reply:
Fixed